applications/API/tests/test_forms.py

```
@@ -0,0 +1,137 @@
+        self.assertTrue(form.is_valid())
+        # clean() should coerce insult_reference_id to the string ref id
+        self.assertEqual(
+            form.cleaned_data["insult_reference_id"], self.insult.reference_id
+        )
+
+    def test_invalid_when_non_anonymous_missing_names(self):
+        """Non-anonymous requires first and last name."""
+        data = self._base_payload(
+            anonymous="", reporter_first_name="", reporter_last_name=""
+        )
+        self.assert_form_invalid_and_error_present(  # noqa
```

Ruff (0.14.1)

Unused blanket `noqa` directive

Remove unused `noqa` directive

(RUF100)